lang/es_ES.json

```diff
@@ -0,0 +1,313 @@
+    "Report Issue - :achievementTitle": "Reportar problema - :achievementTitle",
+    "What sort of issue would you like to report?": "¿Qué tipo de problema te gustaría reportar?",
+    "The achievement type (progression/win/missable) is not correct.": "El tipo de logro (progresión/victoria/perdible) no es correcto.",
+    "There is a spelling or grammatical error in the title or description.": "Hay un error de ortografía o gramática en el título o descripción.",
+    "I have an issue with this achievement that is not described above.": "Tengo un problema con este logro que no se describe arriba.",
+    "I met the requirements, but the achievement did not trigger.": "Cumplí con los requisitos, pero el logro no se activó.",
+    "I unlocked this achievement without meeting the requirements, and then I reset it.": "Desbloqueé este logro sin cumplir con los requisitos, y luego lo reinicié.",
+    "The achievement triggered, but the unlock didn't appear on my profile.": "El logro se activó, pero el desbloqueo no apareció en mi perfil.",
+    "I unlocked this achievement without meeting the requirements.": "Desbloqueé este logro sin cumplir con los requisitos.",
+    "Yes": "Sí",
+    "No": "No",
+    "Either": "Ambos",
+    "System": "Consolas",
+    "Has achievements": "Tiene logros",
+    "Reset": "Reiniciar",
+    ":count of :total game|:count of :total games": ":count de :total juego|:count de :total juegos",
+    ":count game|:count games": ":count juego|:count juegos",
+    ":count selected": ":count seleccionado",
+    "No options found.": "No se encontraron opciones.",
+    "Clear filters": "Limpiar filtros",
+    "Page :count of :total": "Página :count de :total",
+    "Go to first page": "Ir a la primera página",
+    "Go to previous page": "Ir a la página anterior",
+    "Go to next page": "Ir a la página siguiente",
+    "Go to last page": "Ir a la última página",
+    "Search games": "Buscar juegos",
+    "Search games...": "Buscar juegos...",
+    "Type / to focus the search field.": "Escribe / para enfocar el campo de búsqueda.",
+    "View": "Ver",
+    "Toggle columns": "Alternar columnas",
+    "No results.": "Sin resultados.",
+    "Report Issue": "Reportar problema",
+    "Supported Game Files": "Archivos de juegos compatibles",
+    "Supported Game Files - :gameTitle": "Archivos de juegos compatibles - :gameTitle",
+    "Manage Hashes": "Administrar hashes",
+    "This page shows you what ROM hashes are compatible with this game's achievements.": "Esta página te muestra qué hashes de ROM son compatibles con los logros de este juego.",
+    "Hide": "Ocultar",
+    "Want to Play Games": "Juegos que quiero jugar",
+    "Download Patch File": "Descargar archivo de parche",
+    "Settings": "Configuración",
+    "Are you sure you want to reset your avatar to the default? This cannot be reversed.": "¿Estás seguro de que quieres restablecer tu avatar al predeterminado? Esto no se puede deshacer.",
```

Comment:
Hi, I was working on another spanish translation before discovering about you work. So I'll drop my branch since you got here first.

Just a point, what do you think about using `¿Realmente quieres` instead of `¿Estás seguro de que quieres` in all confirmation prompts. That's how I was setting those translations in my branch. The reason behind that choose of words was to prompt a message without distinction of genre and also a bit shorter.

What do you think?

Reply:
Hi @WedgeSparda, I've added the remaining keys after @wescopeland merged the new <Trans> component that includes advanced translation support.

I've incorporated some of the changes you introduced in your PR #2791 because they improved upon my original translation.

Regarding the phrase _Realmente quieres_, I acknowledge it's more concise, but _Estás seguro de que_ might be a better fit. If it appears odd on the website, we can consider altering it. However, I believe this version is more effectively localized.

What are your thoughts? Should we conduct a direct test to see which option performs better on the website?

Reply:
If you see is a better fit, we can go with `Estás seguro de que` my only concern was the genre, but I think we're all used to that message.

Reply:
We can make RA more inclusive; however, this requires consideration in all the texts we translate.

What do you think? Should we change?

Reply:
I would say yes. We should review future translations to match the current ones, not just in this specific topic.

Reply:
Alright, I will make the changes then.

Reply:
Thank you